docs/README.md

```diff
@@ -20,10 +20,24 @@ For more info, see [Directory Structure](https://gohugo.io/getting-started/direc
 
 ## Building
 
+Before starting, you must install the [Learn Theme](https://github.com/docksal/hugo-theme-learn). This can be done by running `git submodule update` within the local repository.
+
+To have submodules pulled automatically in the future set this within your global git configuration.
+
+```
+git config --global submodule.recurse true
+```
+
 Building and previewing docs is done via a docker container.
 
 Run `make serve` and point your browser to http://localhost:1313 to open the site.
 
+If you do not have Docker installed natively and are using VirtualBox, you can run using the `fin` commmand. When accessing, you would use the IP address assigned to your VirtualBox machine (i.e., http://192.168.64.100:1313).
```

Comment:
You can run? I'm not reading this sentence correctly or it's missing something.

Reply:
Yes, that was pretty horrible grammar, thinking of changing to:

If you do not have Docker installed natively but are using VirtualBox for your Docksal projects, the `fin` commmand can be used. When accessing the built project, you would use the IP address assigned to your VirtualBox machine (i.e., http://192.168.64.100:1313).

Reply:
So `fin` can be used with the make command?

Reply:
@achekulaev wasn't thinking either of those. I was just suggesting running:

```
 DOCKER="fin docker" make serve
```

As I have `make` already just didn't have the docker command.

Reply:
Okay now I understood the context. The sentence needs to be revised. "If you don't have Docker then fin command can be used" sounds like `fin` is somehow replacement for Docker.

I would suggest something like:

If you use Docksal via VirtualBox, then `fin docker` command can be used as a replacement for Docker binary e.g. ` DOCKER="fin docker" make serve`..."

Reply:
@sean-e-dietrich I still believe the sentence needs to be revised. I can do it for you if you don't mind.

Reply:
@achekulaev I don't mind at all. It may be easier if you could please.